lib/hammer_cli_foreman/auth_source_ldap.rb

```diff
@@ -30,7 +30,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
           field :account, _('Account Username')
           field :base_dn, _('Base DN')
           field :groups_base, _('Groups base DN')
-          field :use_netgroups, _('Use Netgroups'), Fields::Boolean
```

Comment:
This is still present in the response, should we rather just add `ldap_group_membership`?

We also can mark `use_netgroups` as deprecated in case we're going to drop that:

```ruby
field :ldap_group_membership, _('LDAP group membership')
field :use_netgroups, _('Use Netgroups'), Fields::Boolean, :replaced_by => [_('Account'), _('LDAP group membership')].join('/')
```